vector/src/main/java/im/vector/app/features/home/room/detail/composer/VoiceMessageRecorderView.kt

```diff
@@ -87,6 +88,15 @@ class VoiceMessageRecorderView @JvmOverloads constructor(
         initListeners()
     }
 
+    override fun onVisibilityChanged(changedView: View, visibility: Int) {
+        super.onVisibilityChanged(changedView, visibility)
+        if (changedView == this && visibility == VISIBLE) {
```

Comment:
Why doing this in this method? shouldn't be handled where you look for recordindState?

Reply:
Because if the composer isn't blank we are hiding (making invisible) the mic icon and we need to clear the content description. Visibility is changing but recordingState is the same (NONE).